src/utils/helpers/structure/updateSectionsReviews.ts

```diff
@@ -6,6 +6,18 @@ interface UpdateSectionsReviewsProps {
   reviewResponses: ReviewResponse[]
   reviewer: User
 }
+
+/**
+ * @function updateSectionsReviews
+ * Update section structure to add review to each PageState elements.
+ * After the sections structure is built (by useLoadSectionsStructure)
+ * this utility function combines reviewResponses in the structure
+ * correct location of related response and element.
```

Comment:
Feels like a missing word here, can't quite work out what you mean. "structure TO current location"? "structure WITH current location"? 🤷‍♂️

Reply:
Hahah it makes more sense now. Thank you : )